features/lwipstack/LWIPStack.h

```diff
@@ -95,6 +101,16 @@ class LWIP : public OnboardNetworkStack, private mbed::NonCopyable<LWIP> {
          */
         virtual char *get_ip_address(char *buf, nsapi_size_t buflen);
 
+        /** Copies IP address of the name based network interface to user supplied buffer
+         *
+         * @param    emac       EMAC HAL implementation for this network interface
```

Comment:
no emac param

Reply:
It exists on original Mbed sources. But is incorrect because there is no  emac  param. So is now removed